nav2_dummy_controller/src/dummy_controller.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <chrono>
+#include <ctime>
+#include <thread>
+
+#include "nav2_dummy_controller/dummy_controller.hpp"
+
+using nav2_tasks::TaskStatus;
+using namespace std::chrono_literals;
+
+namespace nav2_dummy_controller
+{
+
+DummyController::DummyController() : Node("DummyController")
+{
+  RCLCPP_INFO(get_logger(), "Initializing DummyController...");
+
+  auto temp_node = std::shared_ptr<rclcpp::Node>(this, [](auto) {});
+
+  vel_pub_ =
```

Comment:
Use a robot object.

Reply:
yes, we need to standarize the output of the controllers, dwb is not using the robot. I've opened an issue: #381